app/models/custom_field.rb

```diff
@@ -77,10 +78,11 @@ def self.loan_questions(field_set = nil)
   #   and optional for all others
   # - If override is true and no records are present, all are optional
   # - If override is false, inherit from parent
+  # - Root nodes effectively have override always true
   # Note, loan type association records are ignored for questions without the 'override_assocations'
   # flag assigned.
   def required_for?(loan)
-    if override_associations
+    if override_associations || !parent_id
```

Comment:
Curious why `parent_id` is being checked directly here. Prefer using closure_tree methods like `child?`.